refnx/reduce/platypusnexus.py

```diff
@@ -498,6 +498,142 @@ def _chopper_values(self, h5data):
         return master, slave, speeds[0] / 60.0, phases[slave - 1]
 
 
+class PolarisedCatalogue(PlatypusCatalogue):
+    """
+    Extract relevant parts of a polarised PLATYPUS NeXus file
+    for reflectometry reduction
+    """
+
+    def __init__(self, h5d):
+        """
+        Extract relevant parts of a polarised PLATYPUS NeXus file
+        for reflectometry reduction
+        Access information via dict access, e.g. cat['pol_flip_freq'].
+
+        Parameters
+        ----------
+        h5d - HDF5 file handle
+        """
+        super(PolarisedCatalogue, self).__init__(h5d)
+        # Is there a magnet?
+        self.ismagnet = False
+        # Is there a cryocooler?
+        self.iscryo = False
+        # Is there a power supply?
+        self.isPS = False
+
+        d = self.cat
+
+        d = self._polarised_flippers(d, h5d)
+        d = self._analyser_flippers(d, h5d)
+        d = self._analyser_params(d, h5d)
+        d = self._check_sample_environments(d, h5d)
+
+    def _polarised_flippers(self, d, h5d):
+        d["pol_flip_freq"] = h5d[
```

Comment:
how much of `_polarised_flippers` is required? I can see how it would be important to know if the flippers are on/off for reduction purposes, but does one need to know the current/frequency/voltage/guide/voltage?

Reply:
For normal usage, these parameters aren't very necessary - would mainly be for debugging if a scan went wrong for an unknown reason or for optimising the flippers by changing the frequency (which is done very rarely). This could be done otherwise in nexus file-reading software and we can trim these off if you prefer to keep the dict to the most important params